ansible/README.md

```diff
@@ -0,0 +1,77 @@
+#Asqatasun Ansible Role
+This Ansible role will install the` Asqatasun Opensource Accessibility` on Ubuntu 12.04/14.04 LTS.
+Before using this role, you can modify the variables or if you wish, you can continue with the defaults that has been provided with the role.
```

Comment:
Perhaps we should only show the `asqatasun_parameters` section, as it is the only one that would be modified by a user. All other parameters should not be modified. 

Why not place this section at the very bottom for the file ? (and add a comment saying something like "please don't modify below")